app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,3 @@
+#comments {
```

Comment:
I don't know if this would be even needed anymore if we agree on the header/footer thing I was saying in my previous comment :)